exercises/concept/logs-logs-logs/.docs/introduction.md

```diff
@@ -88,6 +88,8 @@ for index, char := range myString {
 Since runes can be stored as 1, 2, 3, or 4 bytes, the length of a string may not always equal the number of characters in the string. Use the builtin `len` function to get the length of a string in bytes and the `utf8.RuneCountInString` function to get the number of runes in a string:
 
 ```go
+import "unicode/utf8"
```

Comment:
This import is missing in other places as well, please refer to the information in the automated comment https://github.com/exercism/go/pull/2071#issuecomment-1013920850 (first point).

Reply:
Should be resolved with commit 040e7e0.

(I searched the whole repo for "utf8." and added the import where it made sense.)